### DISCUSSION TITLE
The project is not synchronized if a photo is attached to the data

### DISCUSSION THREAD

**Discussion by SerhiyKSV** [opengisch/QField#2463] [Q&A]
I take a photo with my phone camera and it appears in the point attributes. But when I synchronize the project, the points with the attached photos are not added to the computer. Points without attached photos are added to the computer. What could be the problem? I use a Xiaomi mi 10t phone with a standard camera.

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).

**Comment by suricactus** [comment]
@SerhiyKSV Hey, thanks for reporting. Sorry for the delayed answer. Can you test with the latest beta and/or nighly builds? There was a small issue regarding photo syncrhonizatio that was fixed recently.

Also, please make sure the attachment form fields are configured to use "relative path" on QGIS.

This has already been discussed here:
https://github.com/opengisch/QField/discussions/2460
